cmd/internal/cli/actions_linux.go

```diff
@@ -216,7 +216,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 	currMask := syscall.Umask(0o022)
 	if !NoUmask {
 		// Save the current umask, to be set for the process run in the container
-		// https://github.com/hpcng/singularity/issues/5214
+		// https://github.com/apptainer/singularity/issues5214
```

Comment:
Looks like we just need to add a `/` like so and then we'll be good to merge
```suggestion
		// https://github.com/apptainer/singularity/issues/5214
```

Reply:
done